README.md

```diff
@@ -510,6 +511,43 @@ data:
 You're welcome to raise issues related to feature requests, bugs, or give us general feedback on this project's GitHub Issues page. 
 The SAP BTP service operator project maintainers will respond to the best of their abilities. 
 
+[Back to top](#sap-business-technology-platform-sap-btp-service-operator-for-kubernetes)
+
+## Deleting the Operator
+
+To delete the operator, run the following command:
+`helm uninstall <release name> -n <name space>`
+
+Example: 
+
+ >   ```
+  >   helm uninstall sap-btp-operator -n sap-btp-operator
+
+
+**Note:** All service bindings and service instances are also deleted during this process.
```

Comment:
Although it is deleted during the process, we should add a recommendation to the user to first delete the instances and binding, and to make sure that some of the instances/bindings dont run into an issue.

Reply:
@avilupu  How do they make sure some of the instances and bindings don't run into an issue. What does it mean "run into an issue?"

Reply:
That means they will try to clean the cluster and if some deletion has issues - they will fix it before they will phase it during the pre-delete failure.

Reply:
**Note:** All service bindings and service instances are also deleted during this process.
Make sure you fix any issues that may arise during the deletion of instances and bindings before you delete the operator.